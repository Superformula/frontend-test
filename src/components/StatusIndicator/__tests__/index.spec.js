import React from 'react';
import { mount } from 'enzyme';
import { StatusIndicator } from '../';

describe('StatusIndicator', () => {
  it('should render without failing', () => {
    const wrapper = mount(<StatusIndicator />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render as open', () => {
    const wrapper = mount(<StatusIndicator isOpen />);
    expect(wrapper).toMatchSnapshot();
  });
});
