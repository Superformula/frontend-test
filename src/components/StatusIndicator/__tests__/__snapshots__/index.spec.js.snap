// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusIndicator should render as open 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  width: 8px;
  height: 8px;
  border-radius: 50%;
  margin-right: 4px;
  background-color: #00E8A4;
}

.c2 {
  text-transform: capitalize;
  font-size: 12px;
  font-weight: 400;
  color: #757575;
}

<StatusIndicator
  isOpen={true}
>
  <Wrapper>
    <div
      className="c0"
    >
      <Dot
        isOpen={true}
      >
        <span
          className="c1"
        />
      </Dot>
      <GenericLabel>
        <span
          className="c2"
        >
          open now
        </span>
      </GenericLabel>
    </div>
  </Wrapper>
</StatusIndicator>
`;

exports[`StatusIndicator should render without failing 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  width: 8px;
  height: 8px;
  border-radius: 50%;
  margin-right: 4px;
  background-color: #FF3548;
}

.c2 {
  text-transform: capitalize;
  font-size: 12px;
  font-weight: 400;
  color: #757575;
}

<StatusIndicator>
  <Wrapper>
    <div
      className="c0"
    >
      <Dot>
        <span
          className="c1"
        />
      </Dot>
      <GenericLabel>
        <span
          className="c2"
        >
          closed
        </span>
      </GenericLabel>
    </div>
  </Wrapper>
</StatusIndicator>
`;
